skins/cat17/src/sass/layouts/_introduction.scss

```diff
@@ -39,3 +39,19 @@
 	padding-top: 0;
 }
 
+.introduction-banner {
+	border: 1px solid $brand-primary;
+	padding: 1em 1em 1em 2em;
+	color: #494f51;
```

Comment:
Where does this come from? Does it take inspiration from `$gray-line`?

Reply:
This comes [from Jan](https://phabricator.wikimedia.org/T180690#3822646), who wanted the text to be a bit lighter than the normal text. I've asked in the issue what color values he wants, if we stay in the given palette or not.